app/src/main/res/layout/option_activity.xml

```diff
@@ -1,31 +1,43 @@
-<androidx.drawerlayout.widget.DrawerLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android"
   xmlns:app="http://schemas.android.com/apk/res-auto"
-  xmlns:tools="http://schemas.android.com/tools"
-  android:id="@+id/options_activity_drawer_layout"
-  android:layout_width="match_parent"
-  android:layout_height="match_parent">
+  xmlns:profile="http://schemas.android.com/tools">
 
-  <LinearLayout
-    android:layout_width="match_parent"
-    android:layout_height="match_parent"
-    android:orientation="vertical">
+  <data>
+    <variable
+      name="viewModel"
+      type="org.oppia.app.options.OptionsActivity" />
+  </data>
 
-    <include
-      android:id="@+id/options_activity_toolbar"
-      layout="@layout/toolbar" />
+  <androidx.drawerlayout.widget.DrawerLayout
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/options_activity_drawer_layout"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
 
-    <FrameLayout
```

Comment:
Keep this FrameLayout only instead of `LinearLayout`

Reply:
You don't need to make any change to this file.

Reply:
If I remove the LinearLayout, the toolbar seems to vanish resulting in app crashing.

Reply:
You can check the implementation in `HelpActivity` and `AdministratorControlsActivity`.

Also please tag the reviewer whenever you reply to a comment because otherwise we won't know if you have added a comment. Thanks.